lighthouse-core/scripts/i18n/collect-strings.js

```diff
@@ -103,37 +105,38 @@ function computeDescription(ast, property, value, startRange) {
  *
  * @param {string} message
  * @param {Record<string, string>} examples
- * @return {ICUMessageDefn}
+ * @return {Pick<CtcMessage, 'message'|'placeholders'>}
```

Comment:
```suggestion
 * @return {IncrementalCtc}
```

right?

Reply:
> right?

yeah, takes some changes but probably for the best